`server` service fails to start in latest image

```
/app # gunicorn --bind 0.0.0.0:80 --workers "$GUNICORN_WORKERS" patchman.wsgi:application
[2024-01-01 17:21:57 +0000] [30] [INFO] Starting gunicorn 20.1.0
[2024-01-01 17:21:57 +0000] [30] [INFO] Listening at: http://0.0.0.0:80 (30)
[2024-01-01 17:21:57 +0000] [30] [INFO] Using worker: sync
[2024-01-01 17:21:57 +0000] [31] [INFO] Booting worker with pid: 31
[2024-01-01 17:21:57 +0000] [32] [INFO] Booting worker with pid: 32
[2024-01-01 17:21:58 +0000] [31] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/gunicorn/arbiter.py", line 589, in spawn_worker
    worker.init_process()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/workers/base.py", line 134, in init_process
    self.load_wsgi()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
    return self.load_wsgiapp()
  File "/usr/local/lib/python3.9/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/local/lib/python3.9/site-packages/gunicorn/util.py", line 359, in import_app
    mod = importlib.import_module(module)
  File "/usr/local/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/app/patchman/wsgi.py", line 25, in <module>
    application = get_wsgi_application()
  File "/usr/local/lib/python3.9/site-packages/django/core/wsgi.py", line 13, in get_wsgi_application
    return WSGIHandler()
  File "/usr/local/lib/python3.9/site-packages/django/core/handlers/wsgi.py", line 127, in __init__
    self.load_middleware()
  File "/usr/local/lib/python3.9/site-packages/django/core/handlers/base.py", line 40, in load_middleware
    middleware = import_string(middleware_path)
  File "/usr/local/lib/python3.9/site-packages/django/utils/module_loading.py", line 17, in import_string
    module = import_module(module_path)
  File "/usr/local/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/usr/local/lib/python3.9/site-packages/whitenoise/middleware.py", line 5, in <module>
    from whitenoise.django import DjangoWhiteNoise
  File "/usr/local/lib/python3.9/site-packages/whitenoise/django.py", line 18, in <module>
    from django.utils.six.moves.urllib.parse import urlparse
ModuleNotFoundError: No module named 'django.utils.six'
[2024-01-01 17:21:58 +0000] [31] [INFO] Worker exiting (pid: 31)
[2024-01-01 17:21:58 +0000] [30] [WARNING] Worker with pid 32 was terminated due to signal 15
[2024-01-01 17:21:58 +0000] [30] [INFO] Shutting down: Master
[2024-01-01 17:21:58 +0000] [30] [INFO] Reason: Worker failed to boot.
```

It appears bumping whitenoise (currently pinned to 3.3.1, per recommendation from https://github.com/furlongm/patchman/blob/main/INSTALL.md#virtualenv--pip) to latest (6.6.0 at time of writing) resolves this.

--- a/requirements.txt
+++ b/requirements.txt
@@ -4,4 +4,4 @@ gunicorn==20.1.0
 mysqlclient==2.1.1
 psycopg[binary]==3.1.4
 redis==4.3.4
-whitenoise==3.3.1
+whitenoise==6.6

